### PR TITLE
Update Android job

### PR DESCRIPTION
**Pull request by mario**
Signed-off-by: Mario Danic <mario@lovelyhq.com>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@mario can you rebase please? :) Q: What is the purpose of this PR? Updating the library + necessary changes due to changed API?

**Comment by mario** [comment]
Sure, but not today. And this is indeed the point.

**Comment by tobiasKaminsky** [comment]
Drone says:
>   symbol:   class UploadUtils
  location: package com.owncloud.android.utils
/drone/src/github.com/nextcloud/android/src/main/java/com/owncloud/android/operations/UploadFileOperation.java:51: error: cannot find symbol
import com.owncloud.android.utils.UploadUtils;

Please fix and rebase, then I'll have a look.

**Comment by mario** [comment]
Should be fine now.

**Comment by tobiasKaminsky** [comment]
Unfortunately not, all drone jobs fail.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky Drone is happy now, unfortunately I didn't/don't have the time to test but fixed the lint issues to get the build to turn green...

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky  do you have the time to check this one out? :)

File: src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java [hunk 1]
```diff
@@ -113,11 +114,10 @@ public static void startContactBackupJob(Account account) {
         PersistableBundleCompat bundle = new PersistableBundleCompat();
         bundle.putString(ContactsBackupJob.ACCOUNT, account.name);
 
+        cancelPreviousContactBackupJobForAccount(MainApp.getAppContext(), account);
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java]: The jobs are stopped here. But where is it restarted? 
When enabling the switch, I guess. But what happens if you open up the activity (job is cancelled) and then leave without modifying the switch.

**Comment by tobiasKaminsky** [comment]
The changes on job (execution option, ...) I cannot really judge, but the rest seems fine for me, with only one question left.

File: src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java (hunk 1 above)
Comment (mario) [reply src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java]: Two lines down :) 

https://github.com/nextcloud/android/blob/573c11d7bb8b5b125ff04cf18dbd8999b6d5eeb6/src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java#L119

Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java]: 👍

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky question answered: https://github.com/nextcloud/android/pull/1675#discussion_r154897317

File: src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java (hunk 1 above)
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java]: Ah, somehow I missed it.

**Comment by AndyScherzinger** [comment]
👍 @tobiasKaminsky merge then?

**Comment by tobiasKaminsky** [comment]
👍 
Yes, I'll rebase it and merge it then.

**Pull request merged**